proj/.ocamlinit

```diff
@@ -1,2 +0,0 @@
-#use "topfind";;
-open Printf;;
```

Comment:
The intent of `.ocamlinit` was to have these commands loaded when running `utop` locally. Also, it's a stub letting the user know they can add their own `#require ...`  or `open` directives for working with the project. Now I see opam installs a `~/.ocamlinit` containing `#use "topfind";;`. Is this why you removed it?

Reply:
Yeah I thought this file would be redundant